### PR TITLE
[17.0][MIG] product_packaging_dimension: Migration to 17.0

### PR DESCRIPTION
**Pull request by nguyenminhchien**
Ref: BSRD-720
Depend on:
- [x] https://github.com/OCA/product-attribute/pull/1606

### CODE REVIEW

**Review by ferran-S73 — APPROVED** [review]

File: product_packaging_dimension/README.rst [hunk 1]
```diff
@@ -36,11 +36,6 @@ and volume of product packagings.
 .. contents::
    :local:
 
-Known issues / Roadmap
-======================
-
--  Resolve conflict with delivery module
```
Comment (nguyenminhchien) [inline product_packaging_dimension/README.rst]: conflict resolved by https://github.com/odoo/odoo/commit/1cd0a138cf1288e2bbcad0ad261696b7ff727feb

**Review by leemannd — APPROVED** [review]

**Comment by edy1192** [comment]
Hi @gurneyalex do you have any news about this PR?

**Pull request merged**